client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Can you just add all these under the client section in the root .gitignore, best practice to just have 1